### PR TITLE
New package: mopidy-mpd-3.0.0

### PR DESCRIPTION
**Pull request by Eluminae**
Should I rename it to "Mopidy-MPD" as the python package name ?

### CODE REVIEW

File: srcpkgs/mopidy-mpd/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+archs=noarch
+wrksrc="Mopidy-MPD-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="mopidy python3-pykka python3-setuptools"
+short_desc="Mopidy extension for controlling Mopidy from MPD clients"
+maintainer="Stacy Harper <contact@stacyharper.net>"
+license="Apache-2.0"
+homepage="http://www.mopidy.com"
+changelog="https://github.com/mopidy/mopidy/raw/develop/docs/changelog.rst"
```
Comment (Chocimier) [inline srcpkgs/mopidy-mpd/template]: there is separate repo for -mpd

File: srcpkgs/mopidy-mpd/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'mopidy-mpd'
+pkgname=mopidy-mpd
+version=3.0.0
+revision=1
+archs=noarch
+wrksrc="Mopidy-MPD-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="mopidy python3-pykka python3-setuptools"
```
Comment (Chocimier) [inline srcpkgs/mopidy-mpd/template]: Please add checkdepends="${depends} python3-gobject python3-pytest"

File: srcpkgs/mopidy-mpd/template (hunk 1 above)
Comment (Eluminae) [reply srcpkgs/mopidy-mpd/template]: True! done

File: srcpkgs/mopidy-mpd/template (hunk 2 above)
Comment (Eluminae) [reply srcpkgs/mopidy-mpd/template]: Added

**Pull request merged**